content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1006,6 +1006,27 @@ status:
   terminating: 3 # three Pods are terminating and have not yet reached the Failed phase
 ```
 
+### Delegation of managing a Job object to external controller
+
+{{< feature-state for_k8s_version="v1.30" state="alpha" >}}
+
+This feature allows you to disable the built-in Job controller, for a specific
+Job, and delegate reconciliation of the Job to an external controller.
+
+By setting a custom value of the `spec.managedBy` field - any value
+other than `kubernetes.io/job-controller` - you indicate the controller which
+reconciles the Job. The value of the field is immutable.
+
+{{< note >}}
+When using this feature make sure the controller indicated by the field is
+installed, otherwise the Job may not be reconciled at all.
+{{< /note >}}
+
+{{< note >}}
+When developing an external Job controller make sure it does not use the
```

Comment:
Should we mention that the controllers should conform to the API types, API comments? Not sure if we can define the expected Job behaviour in other way...

And also pass the conformance tests? Or other tests as well?

Reply:
We could say certainly say that the controller status updates should pass the validation rules and be conformant with the API field definitions in the API comments. 

However, running the conformance tests as such might be infeasible. For example in case of MultiKueue multiple conformance tests also assert on pods, which would not pass for the controller, for example here: https://github.com/kubernetes/kubernetes/blob/246e678acc97d68261ce3e9ae858332f01b44058/test/e2e/apps/job.go#L94-L98. Also, I don't think we will have any power to enforce some external controller was validated against the conformance tests. I think the status validation is the tool to enforce that conformance in fact.

 I will update the note.

Reply:
I think, we should say that they should also be conformant with the spec.

I see. Nevertheless, we could write about the compliance with the conformance tests in more generic way. That is if you run the pods somewhere else, you could consider that and update the places in the tests. I suppose you might not even run the pod/executable in kubernetes.

Yes, we do not have any power to enforce this, but if we get a bug that a job is behaving in an unsupported way, we can point to the documentation and say that the external controller should comply.

@alculquicondor @soltysh for additional opinions

Reply:
Updated with some initial formulation. PTAL.

Reply:
lgtm, thanks for updating the section